Update threat-hunting-in-threat-explorer.md (#10364)
MicrosoftDocs/microsoft-365-docs: microsoft-365/security/office-365-security/threat-hunting-in-threat-explorer.md

Defender for Office 365 Plan 1 uses *Real-time detections*, which is a subset of the *Threat Explorer* (also called *Explorer*) hunting tool in Plan 2. In this series of articles, most of the examples were created using the full Threat Explorer. Admins should test any steps in Real-time detections to see where they apply.

> After you go to **Explorer**, by default, you'll arrive on the **All email** page, but use the **tabs** to navigate to the available views. If you're hunting Phish, or digging into a threat campaign, choose those views.

Reviewer: ```suggestion
After you go to **Explorer**, by default, you'll arrive on the **All email** page, but use the tabs to navigate to the available views. If you're hunting phish or digging into a threat campaign, choose those views.
```

> Once a security operations (Sec Ops) person selects the data they want to see, they can further narrow down the data by applying filters like Sender, Recipient, Subject etc. or select appropriate date range to get the desired results. Remember to select Refresh to complete your filtering actions.

Reviewer: ```suggestion
Once a security operations (Sec Ops) person selects the data they want to see, they can further narrow down the data by applying filters such as Sender, Recipient, and Subject,  or select an appropriate date range to get the desired results. Remember to select Refresh to complete your filtering actions.
```

> [!TIP]
> If Sec Ops uses **Tags** to mark accounts they consider high valued targets, they can make selections like *Phish View with a Tags filter focus (include a date range if used)*. This will show them any phishing attempts directed at their high value user targets during a time-range (like dates when certain phishing attacks are happening a lot for their industry).

> With the new version of Threat Explorer, users can use new dropdown options with four new operators on the filters-
>   - Equals any of – returns values matching the exact user input
>   - Equals none of - returns values not matching the exact user input
>   - Contains any of - returns values partially matching user input
>   - Contains none of - returns values not partially matching user input
> Note: The above filter conditions are availble based on filter types and input types.

Reviewer: ```suggestion
With the new version of Threat Explorer, users can use the following new dropdown options with four new operators on the filters:
  - Equals any of – returns values matching the exact user input.
  - Equals none of – returns values not matching the exact user input.
  - Contains any of – returns values partially matching user input.
  - Contains none of – returns values not partially matching user input.

Note that these filter conditions are available based on filter types and input types.
```
